.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # Weekly updates for devDependencies.
+  - package-ecosystem: 'npm'
```

Comment:
this will bump/check only devDeps in root `package.json` correct ?

Reply:
yes!